Update enroll-a-device.md (#8870)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/rooms/enroll-a-device.md

## URLs Required for Communication

 > [!NOTE]

>  > All network traffic between the MTR devices agent and the Microsoft Teams Rooms – Managed Services service portal is SSL over port 443*.*  The below URLs are for managed service agent, for Teams information see [Office 365 URLs and IP address ranges - Microsoft 365 Enterprise | Microsoft Docs](/microsoft-365/enterprise/urls-and-ip-address-ranges?view=o365-worldwide&preserve-view=true).

Reviewer: ```suggestion
 > All network traffic between the MTR devices agent and the Microsoft Teams Rooms – Managed Services service portal is SSL over port 443*.*  The following URLs are for managed service agent; for Teams information see [Office 365 URLs and IP address ranges - Microsoft 365 Enterprise | Microsoft Docs](/microsoft-365/enterprise/urls-and-ip-address-ranges?view=o365-worldwide&preserve-view=true).
```

The following hosts must be allowed if you have **traffic allowlist** enabled within your enterprise environment:

**All Devices**<br>
agent.rooms.microsoft.com<br>
global.azure-devices-provisioning.net<br>
gj3ftstorage.blob.core.windows.net<br>

**APAC**<br>
mmrprodapaciot.azure-devices.net<br>
mmrprodapacstor.blob.core.windows.net<br>

**EMEA**<br>
mmrprodemeaiot.azure-devices.net<br>
mmrprodemeastor.blob.core.windows.net<br>

**NOAM**<br>
mmrprodnoamiot.azure-devices.net<br>
mmrprodnoamstor.blob.core.windows.net
  
  

> The first 3 URLs are required on all devices and are used in initial enrollment and host updates and remediations used by devices. The remaining URLs are based on tenant region. You can identify your tenants region by clickin the '?' in the top right of the portal and looking under 'Region'. These regional URLs are used for telemetry and log collection from the devices. 

Reviewer: ```suggestion
The first three URLs are required on all devices and are used in initial enrollment, host updates, and remediations used by devices. The remaining URLs are based on tenant region. You can identify your tenant's region by selecting the '?' in the top right of the portal and looking under _Region_. These regional URLs are used for telemetry and log collection from the devices. 
```

1. From the left-hand menu in the Microsoft Teams Rooms – Managed Services portal, select **Rooms**.  
1. In the list of rooms provided, choose the room you want to unenroll and select **Unenroll** to stop getting incident alerts or investigation tickets, or to report an incident for the room.

## Troubleshooting
  
Our agent test tool is available at [https://aka.ms/mmr/AgentTestTool](https://aka.ms/mmr/AgentTestTool).

> This will check for some common misconfigurations or blocked network connections and suggest solutions. It will also collect the agent logs, if no issue is identified or you require more assistance please create a customer reported incident within the managed service portal and attach the logs generated by the test tool. 

Reviewer: ```suggestion
This will check for some common misconfigurations or blocked network connections and suggest solutions. It will also collect the agent logs, if no issue is identified or you require more assistance please create a customer-reported incident within the managed service portal and attach the logs generated by the test tool. 
```
